### DISCUSSION TITLE
Docs: explain where you can get the OAUTH2_SECRET

### DISCUSSION THREAD

**Discussion by giancarlosisasi** [vercel/vercel#6947] [Help] [answered]
I was reading this section https://vercel.com/docs/integrations#webhooks/securing-webhooks

There is a paragraph
````
The recommended method to check is to use the x-vercel-signature security header you receive with each request. The value of this header corresponds to the sha1 of the payload body using your client secret.
````

The last part says " using your client secret." but then in the example, it uses the term `OAUTH2_SECRET`. Maybe we can use the same terminology for both values?

Also, the link of `client secret.` is broken, it doesn't redirect to the endpoints/o-auth2 section and it doesn't explain where you can get the OAUTH2_SECRET
https://vercel.com/docs/rest-api#endpoints/o-auth2

**Comment by unr** [comment]
This is still an issue, +1

**Comment by amyegan** [comment, marked as answer]
Thanks for bringing this up! I put in a ticket for the team to take a look and get this updated with the correct information/link.
